test.sh

```diff
@@ -333,7 +333,8 @@ function build-and-run-sample-c-appln-with-LOC-encoding()
 
     local program="./c-sample"
 
-    make clean && L3_LOC_ENABLED=${loc_encoding}  make c-sample
+    L3_LOC_ENABLED=${loc_encoding} make clean
```

Comment:
`L3_LOC_ENABLED` env-var is needed to ensure that `make clean` properly deletes some LOC-related `.o`'s created as part of the build.